src/pages/signup.jsx

```diff
@@ -60,7 +60,7 @@ const SignUp = () => {
       !/[A-Za-z0-9!@#$%^&*()_+]/.test(values.rawPassword) ||
       values.rawPassword.length <= 8
     ) {
-      setErrorMessage("Please fill the details carefully");
+      setErrorMessage( <div className="text-red-500 font-semibold text-sm">Please fill the details carefully</div>);
```

Comment:
```suggestion
      setErrorMessage("Please fill the details carefully");
```

1. please check the indentation of this line using a prettier formatter.
2. check the line [line #231](https://github.com/aminoxix/BoilerplateWebApp/blob/main/src/pages/signup.jsx#L231) & do the necessary changes!